meta-balena-dunfell/recipes-containers/balena/balena_git.bbappend

```diff
@@ -0,0 +1,3 @@
+# Since gcc 7.3.0, compiling mobynit with pie support fails at runtime. We
+# deactivate it until we figure out why that happens.
+MOBYNIT_EXTRA_LDFLAGS_append = " -no-pie"
```

Comment:
The balena recipe no longer compiles mobynit so these is not needed.

Reply:
cool, that append has been removed now